platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/test/HappyPathTest.java

```diff
@@ -1,18 +1,19 @@
 // SPDX-License-Identifier: Apache-2.0
 package org.hiero.otter.test;
 
+import com.swirlds.logging.legacy.LogMarker;
 import java.time.Duration;
+import org.apache.logging.log4j.Level;
 import org.hiero.otter.fixtures.Network;
 import org.hiero.otter.fixtures.OtterTest;
 import org.hiero.otter.fixtures.TestEnvironment;
 import org.hiero.otter.fixtures.TimeManager;
+import org.hiero.otter.fixtures.Validator.LogFilter;
 import org.hiero.otter.fixtures.Validator.Profile;
-import org.junit.jupiter.api.Disabled;
 
 public class HappyPathTest {
 
     @OtterTest
-    @Disabled
```

Comment:
Is this intended? If you think we should run these tests moving forward, we should reduce the time below. If we simulate 10 seconds or 2 minutes should not make a difference.